spec/factories/france_connect_information.rb

```diff
@@ -6,5 +6,9 @@
     birthdate { '1976-02-24' }
     france_connect_particulier_id { '1234567' }
     email_france_connect { 'plip@octo.com' }
+
+    trait :with_user do
+      user { create(:user, email: email_france_connect) }
```

Comment:
Il me semble qu'on peut juste avoir un `build` (plutôt qu'un `create`), et que ActiveRecord persistera automatiquement l'objet associé si jamais on demande un `create(:france_connect_information)`.
 
```suggestion
      user { build(:user, email: email_france_connect) }
```